front/app/api/initiative_action_descriptors/keys.ts

```diff
@@ -1,8 +1,13 @@
-const initiativeActionDescriptorsKeys = {
-  all: () => [{ type: 'initiatives', variant: 'action_descriptor' }],
-  items: () => [
-    { ...initiativeActionDescriptorsKeys.all()[0], operation: 'item' },
-  ],
+import { QueryKeys } from 'utils/cl-react-query/types';
+
+const baseKey = {
+  type: 'initiativs',
```

Comment:
```suggestion
  type: 'initiatives',
```

Reply:
This with the action descriptor is a bit of a weird one, tbh.. I don't think the type `initiatives` is super clear in this case but it's what the back-end returns and it's different from `initiative` which is returned by the actual `/initiatives` endpoint, so it's not critical to change it.